contracts/FlowIDTableStaking.cdc

```diff
@@ -815,6 +825,38 @@ pub contract FlowIDTableStaking {
             FlowIDTableStaking.account.save(false, to: /storage/stakingEnabled)
         }
 
+        /// Removes nodes by setting their weight to zero and refunding
+        /// staked and delegated tokens.
+        access(account) fun removeAndRefundNodeRecord(nodeRecord: &NodeRecord) {
```

Comment:
@joshuahannan Should this be `access(self)` since this is just a helper for `removeUnapprovedNodes` and it takes a reference?